src/Microsoft.Health.Fhir.Liquid.Converter/Cda/CdaDataParser.cs

```diff
@@ -0,0 +1,122 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+using Microsoft.Health.Fhir.Liquid.Converter.Exceptions;
+using Microsoft.Health.Fhir.Liquid.Converter.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.Cda
+{
+    public class CdaDataParser
+    {
+        public IDictionary<string, object> Parse(string document)
+        {
+            try
+            {
+                if (string.IsNullOrEmpty(document))
+                {
+                    throw new DataParseException(FhirConverterErrorCode.NullOrEmptyInput, Resources.NullOrEmptyInput);
+                }
+
+                // Remove line breaks to avoid invalid line breaks in json value
+                document = Regex.Replace(document, @"\r\n?|\n", string.Empty);
```

Comment:
do we have better way to handle line break? just removing these chars may have side effects. e.g. line number in error message may change to invalid.